resources/assets/js/translations/extractedMessages/resources/assets/js/components/ApplicationReview/ReviewApplications.json

```diff
@@ -9,4 +9,4 @@
     "description": "Welcome header on app main page",
     "defaultMessage": "{dayCount, plural, =0 {No Days} one {# Day} other {# Days}} Since Close"
   }
-]
+]
```

Comment:
Not sure how this is happening, since we have `files.insertFinalNewline` turned on in the `.vscode` settings, but we shouldn't be committing files that have no changes or are unrelated to the current issue.